app/assets/javascripts/surveys.js

```diff
@@ -77,7 +77,7 @@
       var surveyUrl = survey.url;
 
       // Survey monkey can record the URL of the survey link if passed through as a query param
-      if (/surveymonkey/.test(surveyUrl)) && (surveyUrl.indexOf('?c=') === -1) {
+      if ((/surveymonkey/.test(surveyUrl)) && (surveyUrl.indexOf('?c=') === -1)) {
         surveyUrl += "?c=" + root.location.pathname;
       }
 
```

Comment:
This commit is here to change the date to make it easier to test on integration and the commit will ultimately be discarded before we merge and deploy (leaving the date at Dec 5th).  However the 2nd part of the commit includes some bracketing that should be included in the previous commit.